### PR TITLE
zh-CN: update glossary/accessibiliity

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

fix typos and update links

### Motivation

missing translaiton for `despite` (causing the meaning changed)

incorrect translation position for word `best`

wikipedia link removal (remove redirect)

### Additional details

image

### Related issues and pull requests

### CODE REVIEW

**Comment by PassionPenguin** [comment]
Plus, got a question. which of the following link is valid? (never have i used links with brackets before..but it seems to work?)

with bracket:

[`- 维基百科上的[抽象化](<https://zh.wikipedia.org/wiki/抽象化_(計算機科學)>)`](https://github.com/mdn/translated-content/blob/740eed14a72a27d0569b87dd8cda815f6c821bf1/files/zh-cn/glossary/abstraction/index.md?plain=1#L40)

or without bracket:

[`- 维基百科的 [Web 无障碍](https://zh.wikipedia.org/wiki/Web_accessibility)词条](https://github.com/mdn/translated-content/blob/740eed14a72a27d0569b87dd8cda815f6c821bf1/files/zh-cn/glossary/accessibility/index.md?plain=1#L13)

**Review by yin1999 — COMMENTED** [review]
nits:

**Comment by yin1999** [comment]
> Plus, got a question. which of the following link is valid? (never have i used links with brackets before..but it seems to work?)
> 
> with bracket:
> 
> [`- 维基百科上的[抽象化](<https://zh.wikipedia.org/wiki/抽象化_(計算機科學)>)`](https://github.com/mdn/translated-content/blob/740eed14a72a27d0569b87dd8cda815f6c821bf1/files/zh-cn/glossary/abstraction/index.md?plain=1#L40)
> 
> or without bracket:
> 
> [`- 维基百科的 [Web 无障碍](https://zh.wikipedia.org/wiki/Web_accessibility)词条`](https://github.com/mdn/translated-content/blob/740eed14a72a27d0569b87dd8cda815f6c821bf1/files/zh-cn/glossary/accessibility/index.md?plain=1#L13)

Both of them is valid format. If the link itself has parentheses, it needs to be wrapped in angle brackets.

File: files/zh-cn/glossary/accessibility/index.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 title: Accessibility（无障碍）
 slug: Glossary/Accessibility
+l10n:
+  sourceCommit: 141ed49ccb29726b8154148a990c4854d7af7d09
 ---
 
 {{GlossarySidebar}}
 
-_无障碍网页_（Web Accessibility，缩写：**A11Y**）指在物理条件和技术条件限制下，保证网站达到最佳可用性的实践。{{Glossary("W3C")}} 组织的 {{Glossary("WAI","Web 无障碍倡议")}}（WAI）正式定义和论述了 Web 无障碍技术。
+**无障碍网页**（Web Accessibility，缩写：**A11Y**）指尽管在物理条件和技术条件限制下，仍能保证网站可用的最佳实践。{{Glossary("W3C")}} 组织的 {{Glossary("WAI","Web 无障碍倡议")}}（WAI）正式定义和论述了 Web 无障碍技术。
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/glossary/accessibility/index.md:10]: Please follow the style of the English document. We have apply the bold style to the word "A11Y".

[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/glossary/accessibility/index.md]: I see

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**